tools/line_endings.py

```diff
@@ -1,37 +1,47 @@
+#!/usr/bin/env python
 from __future__ import print_function
-import sys, shutil, os
+import os
+import sys
+
 
 def convert_line_endings(text, from_eol, to_eol):
-  if from_eol == to_eol: return text
+  if from_eol == to_eol:
+    return text
   return text.replace(from_eol, to_eol)
 
-# if dst_filename is empty, operates in place on src_filename
+
```

Comment:
Double empty lines after a function has not been a common convention (though don't mind much either way)